Chaincase.UI/ViewModels/LoadWalletViewModel.cs.orig

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Chaincase;
+using Chaincase.Common;
+using Chaincase.Common.Contracts;
+using Chaincase.Common.Services;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using ReactiveUI;
+using Splat;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Helpers;
+using WalletWasabi.Logging;
+using WalletWasabi.Wallets;
+
+namespace Chaincase.UI.ViewModels
+{
+    public class LoadWalletViewModel : ReactiveObject
+    {
+        private readonly WalletManager _walletManager;
+        private readonly Config _config;
+        private readonly UiConfig _uiConfig;
+        private readonly SensitiveStorage _storage;
+
+        private string _password;
+        private string _seedWords;
+
+        private readonly string ACCOUNT_KEY_PATH = $"m/{KeyManager.DefaultAccountKeyPath}";
+        private const int MIN_GAP_LIMIT = KeyManager.AbsoluteMinGapLimit * 4;
+
+<<<<<<< HEAD
```

Comment:
🤔

Reply:
oh come on you don't like my recently added orig files